src/tools/rbd/action/Bench.cc

```diff
@@ -383,11 +383,13 @@ int do_bench(librbd::Image& image, io_type_t io_type,
       cur_off = 0;
 
       double time_sum = boost::accumulators::rolling_sum(time_acc);
-      printf("%5d  %8d  %8.2lf  %8.2lf\n",
+      printf("%5d  %8d  %8.2lf",
              (int)elapsed.count(),
              (int)(ios - io_threads),
-             boost::accumulators::rolling_sum(ios_acc) / time_sum,
-             boost::accumulators::rolling_sum(off_acc) / time_sum);
+             boost::accumulators::rolling_sum(ios_acc) / time_sum);
+      std::cout.width(10);
+      std::cout << byte_u_t(boost::accumulators::rolling_sum(off_acc) / time_sum) << "/s"
```

Comment:
Avoid mixing and matching C and C++ styles -- if you want to help change this over to C++, can you do the whole output line?